.drone.yml

```diff
@@ -5,17 +5,18 @@ type: kubernetes
 name: build
 
 steps:
-- name: build project
-  image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
-  commands:
-  - ./gradlew assemble --no-daemon
+  - name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - ./gradlew downloadSchema
```

Comment:
Could you make downloadSchema a dependancy of build/assemble.
That would mean devs don't need to run it locally and forget to pull it?